internal/awx/resource_job_template_launch.go

```diff
@@ -129,6 +129,7 @@ func resourceJobTemplateLaunchCreate(ctx context.Context, d *schema.ResourceData
 	if d.Get("wait_for_completion").(bool) {
 		err = jobTemplateLaunchWait(ctx, client.JobService, res, d.Timeout(schema.TimeoutCreate))
 		if err != nil {
+			//nolint:govet
 			return utils.Diagf(
 				"JobTemplate execution failure",
 				fmt.Sprintf("JobTemplateLaunch with ID %d and template ID %d, failed to complete %s", res.ID, d.Get("job_template_id").(int), err.Error()),
```

Comment:
you can just do this to avoid the issue with the linter
```suggestion
				"JobTemplateLaunch with ID %d and template ID %d, failed to complete %s", res.ID, d.Get("job_template_id").(int), err.Error(),
```